### PR TITLE
chore(vector_rs): add docstrings for functions

### PR DESCRIPTION
**Pull request by naschidaniel**

### CODE REVIEW

**Comment by naschidaniel** [comment]
@hiaselhans,

i added some descriptions for main functions.

File: src/vector.rs [hunk 1]
```diff
@@ -24,28 +24,52 @@ impl Vector2D {
         Ok(Self {v})
     }
 
+    /// angle($self)
+    /// --
+    ///
+    /// This function calculates the angle between x and y from a Vector2D.
```
Comment (hiaselhans) [inline src/vector.rs]: "angle relative to the x-axis" ?

File: src/vector.rs [hunk 2]
```diff
@@ -24,28 +24,52 @@ impl Vector2D {
...
+    /// This function copies a Vector2D object.
     pub fn copy(&self) -> Self {
         let v = self.v.clone();
         Self {v}
     }
 
+    /// cross($self, other)
+    /// --
+    ///
+    /// This function calculates the cross product of a Vector2D.
```
Comment (hiaselhans) [inline src/vector.rs]: two! :)

File: src/vector.rs [hunk 3]
```diff
@@ -24,28 +24,52 @@ impl Vector2D {
...
+    /// This function calculates a normalized Vector2D.
     pub fn normalized(&self) -> Self {
         let v = self.v / self.v.norm();
         Self {v}
     }
 
+    /// length($self)
+    /// --
+    ///
+    /// This function calculates length of a Vector2D.
```
Comment (hiaselhans) [inline src/vector.rs]: the!

**Review by hiaselhans — COMMENTED** [review]
sehr cool!

**Pull request merged**